docs/manifest.json

```
@@ -696,6 +696,11 @@
 					"description": "Integrate Coder with JFrog Artifactory",
 					"path": "./admin/integrations/jfrog-artifactory.md"
 				},
+				{
+					"title": "Istio Integration",
```

```suggestion
					"title": "Integrate Coder with Istio",
```

to match the ones around it